### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by Nerzal** [sv-tools/openapi#44] [General]
Just wanted to tell you, that this lib is a life safer for me! 
Thank you! 

I can finally implement an OpenAPI 3.1.0 spec in [swaggo](https://github.com/swaggo/swag). 
Already started it, it's working perfectly fine so far. 

If i find anything missing, i'll open a PR :)

**Comment by SVilgelm** [comment]
you are welcome, but this is so far from the readiness :-) It is not developer oriented, I thought about rewriting it to avoid those infinity `Extandable[RefOrSpec[...` calls

anyway I'm glad that you can use it, feel free to open any PRs
